2026-02-16 AEW proposal (#186)
makerdao/next-gen-atlas: Sky Atlas/Sky Atlas.md

[Cryptographically signed AD Recognition Submission Message from the Ethereum address controlling Delegate Contract]
[Cryptographically signed AD Recognition Submission Message from the Ecosystem Actor Ethereum address, where this address is not an address controlling a Delegate Contract]

#### A.1.5.1.4 - Probationary Period [Core]  <!-- UUID: 66183cd4-0cf7-4ba5-bfde-373e50296cf6 -->

> The Core Facilitator applies heightened accountability standards to Aligned Delegates during their first 90 days following recognition (the “Probationary Period”) as specified in the documents herein.

Reviewer: ```suggestion
The Core Facilitator applies heightened accountability standards to Aligned Delegates during their first 90 days following recognition (the "Probationary Period") as specified in the documents herein.
```

The Core Facilitator may issue a Tier 1 (Procedural) notice to an Aligned Delegate whose pattern of abstentions indicates disengagement from active governance participation. In evaluating whether abstentions are excessive, the Core Facilitator should consider:

1. the proportion of abstentions relative to total votes cast over the preceding six (6) months;
2. whether the Aligned Delegate provided reasoning for abstentions; and
3. whether the pattern suggests avoidance of accountability rather than conflict of interest or insufficient expertise.

This provision does not establish a specific numerical threshold for excessive abstentions. Rather, the Core Facilitator exercises reasonable judgment in identifying patterns inconsistent with the active stewardship expected of Aligned Delegates.

#### A.1.5.2.2 - Aligned Delegate Communication Responsibilities [Core]  <!-- UUID: 3a8c2b92-3a23-48a0-87c4-ea1325bbbd98 -->

Aligned Delegates must provide substantive reasoning when explaining their votes. A compliant vote explanation must:

> 1. Demonstrate understanding of the proposal’s core mechanisms or changes;

Reviewer: ```suggestion
1. Demonstrate understanding of the proposal's core mechanisms or changes;
```

##### A.1.5.6.1.3 - Aligned Delegate Breach Record [Active Data Controller]  <!-- UUID: 32862df8-a277-48a3-bb0f-40ed1e051dfd -->

Notices and warnings issued under the Graduated Response Framework are recorded by adding an entry to the Active Data list in [A.1.5.6.1.3.0.6.1 - Aligned Delegate Breach Registry](1ddd9cf6-3f93-4a33-8c1d-80405eec1ffb).

The Active Data is updated as follows:

- The Responsible Party is the Core Facilitator.
- The Update Process must follow the protocol for 'Alignment Conserver Changes'.

###### A.1.5.6.1.3.0.6.1 - Aligned Delegate Breach Registry [Active Data]  <!-- UUID: 1ddd9cf6-3f93-4a33-8c1d-80405eec1ffb -->

Breaches of the Graduated Response Framework are recorded herein.

#### A.1.5.6.2 - Voting Estoppel Rule [Core]  <!-- UUID: c1a5d22a-c30c-4445-80eb-caf3bd908d28 -->

> An Aligned Delegate who votes in favor of a proposal is deemed to have read and understood the key governance implications of that proposal at the time of voting. This creates a bright-line test: the act of voting in favor constitutes affirmative acknowledgment of the proposal’s contents and consequences.

Reviewer: ```suggestion
An Aligned Delegate who votes in favor of a proposal is deemed to have read and understood the key governance implications of that proposal at the time of voting. This creates a bright-line test: the act of voting in favor constitutes affirmative acknowledgment of the proposal's contents and consequences.
```

The materials associated with preparing the Artifact Edit Proposal during the Invocation of this Instance are contained herein.

###### A.6.1.1.8.2.2.2.2.3 - Completed Instances [Core]  <!-- UUID: 2ce26124-5604-42ae-a3ab-9790a4a9a702 -->

The Instances of the Root Edit Primitive with `Completed` Status are contained herein.

###### A.6.1.1.8.2.2.2.2.4 - In Progress Invocations [Core]  <!-- UUID: 719e16f2-bd1b-4ee3-b86b-7e02de533ac5 -->

Because the Root Edit Primitive is deployed only once, no further Instances of the Primitive can be Invoked.

###### A.6.1.1.8.2.2.3 - Light Agent Primitive [Core]  <!-- UUID: 103c14cc-c21d-403b-996e-38f661a63093 -->

The documents herein contain all data and specifications for Launch Agent 7's Instances of the Light Agent Primitive. See [A.2.2.5.3 - Light Agent Primitive](44028423-2cd1-40cb-89ac-3f762b602b90).

###### A.6.1.1.8.2.2.3.1 - Primitive Hub Document [Core]  <!-- UUID: c45b24ee-844e-4fa4-bdd5-6f4af842fbea -->

The documents herein organize all base information relevant to Launch Agent 7's usage of the Light Agent Primitive.

> ###### A.6.1.1.8.2.2.3.1.1 - Glovbal Activation Status [Core]  <!-- UUID: 8d69c6f7-299f-470f-97fd-ba16bc3077b6 -->

Reviewer: ```suggestion
###### A.6.1.1.8.2.2.3.1.1 - Global Activation Status [Core]  <!-- UUID: 8d69c6f7-299f-470f-97fd-ba16bc3077b6 -->
```

The subtrees for failed Invocations of the Distribution Requirement Primitive are stored here.

###### A.6.1.1.8.2.3.1.1.5.1.2 - Suspended Instances [Core]  <!-- UUID: bddcc7eb-5d2d-402b-a2b7-dc8e2be70e1c -->

The subtrees for Instances of the Distribution Requirement Primitive with `Suspended` Status are stored here.

###### A.6.1.1.8.2.3.1.2 - Active Instances [Core]  <!-- UUID: f2c71db2-efde-4d80-800b-945f8ca1dab0 -->

The Instances of the Distribution Requirement Primitive with `Active` Status are stored herein.

###### A.6.1.1.8.2.3.1.2.1 - Single Instance Configuration Document [Core]  <!-- UUID: 2fa1a658-3588-45cd-a6e4-ceb0cc8f57d8 -->

The documents herein contain the Instance Configuration Document for the Single Distribution Requirement Primitive Instance.

###### A.6.1.1.8.2.3.1.2.1.1 - Parameters [Core]  <!-- UUID: 9f1da224-aad0-4903-8e74-7dbd1dc6c30e -->

The documents herein define the parameters of the Single Instance of the Distribution Requirement Primitive.

###### A.6.1.1.8.2.3.1.2.1.1.1 - Terms [Core]  <!-- UUID: b5a5b8d8-7022-46a7-a351-dc767a5618c4 -->

Launch Agent 7 will buy back and distribute 0.25% of its total token supply per year.

> ###### A.6.1.1.8.2.3.1.2.1.1.2 - Custom Instance Paramters [Core]  <!-- UUID: f5b52d8f-a131-428a-bc3b-bd87d2493ecb -->

Reviewer: ```suggestion
###### A.6.1.1.8.2.3.1.2.1.1.2 - Custom Instance Parameters [Core]  <!-- UUID: f5b52d8f-a131-428a-bc3b-bd87d2493ecb -->
```

The documents herein contain all data and specifications for Launch Agent 7's Instances of the Junior Risk Capital Rental Primitive. See [A.2.2.9.2 - Junior Risk Capital Rental Primitive](d8086dc0-7e77-4c6b-98c7-5fc41337a1ce).

###### A.6.1.1.8.2.6.2.1 - Primitive Hub Document [Core]  <!-- UUID: 796d1eab-a018-4e8f-9d86-8e1f2eea9c5b -->

The documents herein organize all base information relevant to Launch Agent 7's usage of the Junior Risk Capital Rental Primitive.

###### A.6.1.1.8.2.6.2.1.1 - Global Activation Status [Core]  <!-- UUID: 24340f8a-a4a7-4002-9c61-c5d7d4e9d4b3 -->

`Inactive`

###### A.6.1.1.8.2.6.2.1.2 - Active Instances Directory [Core]  <!-- UUID: 56655922-1c77-40f7-83f4-3b9af9dc0c59 -->

This document contains a Directory of all Instances of the Junior Risk Capital Rental Primitive with Instance status of `Active`.

###### A.6.1.1.8.2.6.2.1.3 - Completed Instances Directory [Core]  <!-- UUID: efae8d35-a014-41fa-8a82-b2a4488e7058 -->

This document contains a Directory of all Instances of the Junior Risk Capital Rental Primitive with Instance status of `Completed`.

> ###### A.6.1.1.8.2.6.2.1.4 - In Progress Invocations Director [Core]  <!-- UUID: 86ca0f50-a33b-4068-abd4-3f6a3adb9874 -->

Reviewer: ```suggestion
###### A.6.1.1.8.2.6.2.1.4 - In Progress Invocations Directory [Core]  <!-- UUID: 86ca0f50-a33b-4068-abd4-3f6a3adb9874 -->
```

###### A.6.1.1.8.2.6.2.3 - Completed Instances [Core]  <!-- UUID: f12949d0-1f1d-473f-89b1-f833523345af -->

The Instances of the Junior Risk Capital Rental Primitive with `Completed` Status are stored herein.

###### A.6.1.1.8.2.6.2.4 - In Progress Invocations [Core]  <!-- UUID: f2d93d71-1bf9-4199-8190-e1e0beeb5369 -->

The in progress Invocations of the Junior Risk Capital Rental Primitive are contained herein. Once an Invocation is successfully completed, its subtree will be moved to [A.6.1.1.8.2.6.2.2 - Active Instances](9a3c679d-a269-440e-9069-4bd85b428b3f).

###### A.6.1.1.8.2.6.3 - Asset Liability Management Rental Primitive [Core]  <!-- UUID: 52c0af9a-b4e2-4ff5-ad4c-ffb82263d512 -->

The documents herein contain all data and specifications for Launch Agent 7's Instances of the Asset Liability Management Rental Primitive. See [A.2.2.9.3 - Asset Liability Management Rental Primitive](bd1f1ce5-6c31-42fc-a2aa-694acf5eb08c).

###### A.6.1.1.8.2.6.3.1 - Primitive Hub Document [Core]  <!-- UUID: 3f05c2bb-7f5d-4716-b37e-fdda16d2417e -->

The documents herein organize all base information relevant to Launch Agent 7's usage of the Asset Liability Management Rental Primitive.

> ###### A.6.1.1.8.2.6.3.1.1 - Global Activation Status

Reviewer: Missing UUID
